app/controllers/TeamController.js

```diff
@@ -0,0 +1,65 @@
+const TeamService = require("../services/TeamService");
+
+exports.create = (req, res, next) => {
+  TeamService.create({ name: req.body.name })
+    .then((team) => {
+      res.status(201).json(team);
+    })
+    .catch((error) => {
+      next(error);
```

Comment:
Bilo bi dobro da u zavisnosti od toga koji se error dogodio vratis odgovarajuci status kod i poruku. Posto samo to Nikoli pokaziva, najbolje da pogledas kod njega.
https://git.vegaitsourcing.rs/nikola.petrovic/nodejs-learning
Metoda za dodavanje korisnika u tim je dobar primer.